src/java.base/share/classes/java/security/Security.java

```diff
@@ -1,3 +1,4 @@
+/*[INCLUDE-IF JAVA_SPEC_VERSION >= 8]*/
```

Comment:
This line is not necessary: please remove it.

Reply:
Created https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/511
W/ this change, the `INCLUDE-IF` can be removed.

Reply:
done